Client/src/Components/ListRouter.tsx

```diff
@@ -0,0 +1,101 @@
+import { List, ListItem, ListItemIcon, ListItemText, Paper, makeStyles } from "@material-ui/core"
+import { Link as RouterLink, LinkProps as RouterLinkProps } from 'react-router-dom'
+import { aboutRoute, dataCellAnalysisRoute, datasetUploadRoute, fileUploadRoute, graphRoute, homeRoute, profileRoute, researchPaperAnalysisRoute, searchRoute, signInRoute, signUpRoute } from '../Consts/Routes'
+
+import { AboutView } from "./Home/AboutView"
+import AccountBoxIcon from '@material-ui/icons/AccountBox'
+import BarChartIcon from '@material-ui/icons/BarChart'
+import CloudUploadIcon from '@material-ui/icons/CloudUpload'
+import { DataCellAnalysisView } from "./DataCellAnalysis/DataCellAnalysisView"
+import { DatasetUploadView } from "./DatasetUpload/DatasetUploadView"
+import DonutSmallIcon from '@material-ui/icons/DonutSmall'
+import FileUploadView from "./DataCell/FileUploadView"
+import GraphView from "./Graph/GraphView"
+import HomeIcon from '@material-ui/icons/Home'
+import HomeView from "./Home/HomeView"
+import ImageSearchIcon from '@material-ui/icons/ImageSearch'
+import InfoIcon from '@material-ui/icons/Info'
+import PersonIcon from '@material-ui/icons/Person'
+import { ProfileView } from "../Views/ProfileView"
+import React from 'react'
+import { ResearchPaperAnalysisView } from "./ResearchPaperAnalysis/ResearchPaperAnalysisView"
+import { Route } from 'react-router'
+import SearchIcon from '@material-ui/icons/Search'
+import SearchView from "./Search/SearchView"
+import SignIn from "../Authentication/SignIn"
+import SignUp from "../Authentication/SignUp"
+
+interface IProps {
+  icon?: React.ReactElement;
+  primary: string;
+  to: string;
+}
+
+export const linkWidth: number = 240
+
+export const ListItemLink = (props: IProps) => {
+  const { icon, primary, to } = props
+
+  const renderLink = React.useMemo(
+    () =>
+      // eslint-disable-next-line react/display-name
```

Comment:
Is there a reason for having this commented? Testing purposes? Can we delete this line if it's useless?

Reply:
yeah you right, this comment disables eslint rule just for the next line, however I can disable this rule for the whole project, ill remove it

Reply:
it was removed, however not showing up in the conversation section of the PR review, take a look at the Files changed -> my last commit

Reply:
Great thanks!